.bazelci/presubmit.yml

```diff
@@ -149,10 +149,8 @@ tasks:
       - "//..."
       # TODO: There are windows specific build issues in the generated
       # depednencies. This should be fixed and all testing enabled.
-      - "-//basic:test"
       - "-//has_aliased_deps/..."
       - "-//uses_proc_macro/..."
-      - "-//uses_sys_crate/..."
     build_targets: *crate_universe_windows_targets
     # TODO: fix test targets in `crate_universe_windows_targets`
     # test_targets: *crate_universe_windows_targets
```

Comment:
This should be uncommented too if you wanna see the failures from the test target.